cli/cmd/sync.go

```diff
@@ -281,3 +297,14 @@ func sync(cmd *cobra.Command, args []string) error {
 
 	return nil
 }
+
+func pluginEnv(environ []string, pluginName, kind string) []string {
+	env := make([]string, 0)
+	prefix := "__" + kind + "_" + pluginName + "__"
```

Comment:
Technically you can still have the same plugin and kind from different teams on a single sync...

Reply:
Assuming I didn't make a mistake, this should be the `name` field in the plugin spec, so you can't have two of the same